src/playlist-maker.js

```diff
@@ -22,144 +30,175 @@ var indexInSources = function(arr, src) {
   return -1;
 };
 
-// factory method to return a new playlist with the following API
```

Comment:
On third look, the "documentation" project does some weird stuff with JSDoc, completely flattening object/module stucture/membership, which seems like an incredibly poor decision to me.